catalog/package-lock.json

```diff
@@ -5888,8 +5888,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
it's strange that npm doing this again. which version are you using?

Reply:
I'm on 6.4.1. It never stopped doing this weirdness.